### ISSUE TITLE
Missing labels for stories system layer buttons

### ISSUE THREAD

**Issue by newmuis** [ampproject/amphtml#24802]
System layer buttons (share, speaker, (i) icon, sidebar icon, spinner) are all missing labels, so they are verbalized as "button" when read out by screen readers.
